src/EventBus/SubscriberSpecificDomainEvent.php

```diff
@@ -52,7 +52,6 @@ public function getLoggingContext() : array
 	{
 		$result = [
 			'subscriberType' => $this->subscriberType,
-			'eventType' => get_class($this->event),
```

Comment:
`eventType` is now resolved by `MessageContextResolver`.

Reply:
`MessageContextResolver` does not properly support `SubscriberSpecificDomainEvent`:
1) It always returns `'SubscriberSpecificDomainEvent'` instead of e.g. `'FooHappened'` as `eventType`.
2) It calculates `eventHash` from instance of `SubscriberSpecificDomainEvent` instead of the underlying event instance.